src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -135,11 +138,45 @@
     private final LongSupplier supplyTraceId;
 
     private final Long2ObjectHashMap<MqttServer.MqttServerStream> correlations;
-    private final MessageFunction<RouteFW> wrapRoute;
+    private final MessageFunction<RouteFW> wrapRoute = (msgTypeId, buffer, index, length) ->
+            routeRO.wrap(buffer, index, index + length);
```

Comment:
Use `(t, b, i, l)` for brevity instead of `(msgTypeId, buffer, index, length)` in lambda expression.